multiqc/modules/pangolin/pangolin.py

```diff
@@ -103,6 +104,9 @@ def add_version_not_none(version, sample, name):
                 # Avoid generic header ID that clashes with other modules
                 if "qc_status" not in row:
                     row["qc_status"] = row.pop("status")
+                if "qc_notes" in row:
+                    row["qc_notes"] = _format_qc_notes(row["qc_notes"])
+
```

Comment:
I moved parsing the QC notes here, so the parsing errors would more clearly be marked and cought as happening in this module, rather than in the code plotting module.

Reply:
Thanks!